docs/Usage.rst

```diff
@@ -45,7 +45,7 @@ For that, the repository must be cloned and installed:
 
 .. code-block::
 
-   git clone https://github.com/cda-tum/MQTBench.git
+   git clone https://github.com/munich-quantum-toolkit/bench.git
    cd MQTBench
```

Comment:
This could be nice as 
```
git clone https://github.com/munich-quantum-toolkit/bench.git mqt-bench
cd mqt-bench
```
The `cd` line definitely needs fixing either way.